Formula/lib/libdicom.rb

```diff
@@ -0,0 +1,25 @@
+class Libdicom < Formula
+  desc "DICOM WSI read library"
+  homepage "https://github.com/ImagingDataCommons/libdicom"
+  url "https://github.com/ImagingDataCommons/libdicom/releases/download/v1.0.1/libdicom-1.0.1.tar.xz"
+  sha256 "45edf650e214fa4dc905660aa25970210fe851d51fec5145dc369d447af4f3e1"
+  license "MIT"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  # libdicom has a bundled uthash, but use the homebrew one in preference
```

Comment:
would there be some build option corresponding to this comment?

```suggestion
  # libdicom has a bundled uthash, but use the homebrew one in preference
```

Reply:
👍